server/src/utils.ts

```diff
@@ -180,6 +180,31 @@ export let replaceFileExtension = (filePath: string, ext: string): string => {
   return path.format({ dir: path.dirname(filePath), name, ext });
 };
 
+export const toCamelCase = (text: string): string => {
+  return text
+    .replace(/(?:^\w|[A-Z]|\b\w)/g, (s: string) => s.toUpperCase())
+    .replace(/(\s|-)+/g, "");
+};
+
+export const getNamespaceNameFromBsConfig = (
+  projDir: p.DocumentUri
+): string | null => {
+  let bsconfigFile = fs.readFileSync(
+    path.join(projDir, c.bsconfigPartialPath),
+    { encoding: "utf-8" }
+  );
+
+  let bsconfig = JSON.parse(bsconfigFile);
```

Comment:
I don't have a particular suggestion here, just noticing that the compiler accepts comments in `bsconfig.json`, which will cause a parse error here.
Perhaps just avoid crashing the extension would be OK.

Reply:
I made it a bit more user-friendly and show a separate message when the JSON cannot be parsed.
I tested it out on a bsconfig with comment and it shows the error message as expected.